packages/@ourworldindata/grapher/src/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -167,12 +167,22 @@ export class StackedDiscreteBarChart
         return !this.manager.hideLegend
     }
 
+    @computed private get labelFontSize(): number {
+        const innerBounds = this.bounds.padTop(
+            this.showLegend && this.legend.height > 0
+                ? this.legend.height + this.legendPaddingTop
+                : 0
+        )
+        const barHeight = (0.8 * innerBounds.height) / this.items.length
```

Comment:
This is messy but using the computed `barHeight` property doesn't work here because that would result in a MobX cyclic dependency

Reply:
Right; that's a bummer. Should add a comment for that one.

The alternative would be to add an intermediate `boundsWithoutLegend` property (also not very clean, but maybe less likely to go out of sync in the future)...